ast/ArgParsing.cc

```diff
@@ -54,4 +55,30 @@ vector<ParsedArg> ArgParsing::parseArgs(core::Context ctx, ast::MethodDef::ARGS_
     return parsedArgs;
 }
 
+u4 ArgParsing::hashArgs(core::Context ctx, std::vector<ParsedArg> &args) {
```

Comment:
Can we make it return `vector<u4>` where every entry is a precise data that can't have collisions?

Reply:
this will also make it not hashing `shortName` that is a footgun on its own